site-metadata.js

```diff
@@ -113,6 +114,7 @@ siteMetadata.urls.navCommunityLinks = [
   { title: 'Get involved', href: siteMetadata.urls.community, isGatsby: true },
   { title: 'Use cases', href: siteMetadata.urls.useCases, isGatsby: true },
   { title: 'Support', href: siteMetadata.urls.support, isGatsby: true },
+  { title: 'Telemetry', href: siteMetadata.urls.telemetry, isGatsby: true },
```

Comment:
Adding this here will add the link to both the Community nav menu in the Header and the Storybook section of the Footer. Preview here: https://deploy-preview-359--storybook-frontpage.netlify.app/

Is that desired? /cc @winkerVSbecks

Reply:
How about adding it to the footer directly (not via the site metadata). So either it won't show up in tutorials or blog for now or you could patch those footers too. I'm fine with either of those options.

Reply:
Done.

- https://github.com/chromaui/blog/pull/89
- https://github.com/chromaui/learnstorybook.com/pull/596

Reply:
Thanks!